packages/ui/components/form/select/Select.tsx

```diff
@@ -195,3 +205,45 @@ export function SelectWithValidation<
     </div>
   );
 }
+
+export type Option = {
+  value: string;
+  label: string;
+};
+
+const InputOption: React.FC<OptionProps<Option, boolean, GroupBase<any>>> = ({
+  isDisabled,
+  isFocused,
+  isSelected,
+  children,
+  innerProps,
+  className,
+  ...rest
+}) => {
+  const props = {
+    ...innerProps,
+  };
+
+  return (
+    <components.Option
+      {...rest}
+      isDisabled={isDisabled}
+      isFocused={isFocused}
+      isSelected={isSelected}
+      innerProps={props}>
+      <input
+        type="checkbox"
+        className="text-primary-600 focus:ring-primary-500 border-default h-4 w-4 rounded ltr:mr-2 rtl:ml-2"
+        checked={isSelected}
+        readOnly
+      />
+      {children}
+    </components.Option>
+  );
+};
+
+const MultiValue = ({ index, getValue }: { index: number; getValue: any }) => {
+  const { t } = useLocale();
+
+  return <>{!index && <div>{t("nr_event_type", { count: getValue().length })}</div>}</>;
```

Comment:
```suggestion
  return <>{!index && <div>{t("nr_event_type", { count: getValue().length })}</div>}</>;
```

We should handle translations for event_types outside of this component. And pass in the translation. 

This select could be used in parts of the app that dont know what translations are / are not involved with event types

Reply:
Hey @sean-brydon Thanks a lot for the input and I have pushed the fix that you suggested to reduce bundle size.
As for this one, could you please suggest how exactly you want me to pass as a prop?
Like should I pass a boolean to let Select component know whether to render this or not or directly pass the `additionalComponent` as an optional prop?

Also, translations were already being used in `SelectField` so that's why I though it would be okay.
<img width="1091" alt="Screenshot 2023-05-06 at 10 10 17 PM" src="https://user-images.githubusercontent.com/52901335/236636682-489398d0-d7b1-4649-8bb1-1102c1b9472c.png">